services/web/server/tests/unit/with_dbs/03/resource_usage/conftest.py

```diff
@@ -0,0 +1,83 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+from copy import deepcopy
+from typing import Callable
+
+import pytest
+from servicelib.aiohttp.application import create_safe_application
+from simcore_service_webserver.application_settings import setup_settings
+from simcore_service_webserver.db.plugin import setup_db
+from simcore_service_webserver.director_v2.plugin import setup_director_v2
+from simcore_service_webserver.login.plugin import setup_login
+from simcore_service_webserver.products.plugin import setup_products
+from simcore_service_webserver.projects.plugin import setup_projects
+from simcore_service_webserver.resource_manager.plugin import setup_resource_manager
+from simcore_service_webserver.resource_usage.plugin import setup_resource_tracker
+from simcore_service_webserver.rest.plugin import setup_rest
+from simcore_service_webserver.security.plugin import setup_security
+from simcore_service_webserver.session.plugin import setup_session
+from simcore_service_webserver.socketio.plugin import setup_socketio
+from simcore_service_webserver.tags.plugin import setup_tags
+from simcore_service_webserver.wallets.plugin import setup_wallets
+
+API_VERSION = "v0"
+RESOURCE_NAME = "projects"
+API_PREFIX = "/" + API_VERSION
+
+
+DEFAULT_GARBAGE_COLLECTOR_INTERVAL_SECONDS: int = 3
+DEFAULT_GARBAGE_COLLECTOR_DELETION_TIMEOUT_SECONDS: int = 3
+
+
+@pytest.fixture
+def client(
+    event_loop,
+    aiohttp_client,
+    app_cfg,
+    postgres_db,
+    mocked_director_v2_api,
+    mock_orphaned_services,
+    redis_client,  # this ensure redis is properly cleaned
+    monkeypatch_setenv_from_app_config: Callable,
+):
+    # config app
+    cfg = deepcopy(app_cfg)
```

Comment:
This is the old way to create clients! Avoid eveyrthing regarding ``app_cfg`` and use instead the env-variables to configure your service. The idea is to override `app_environment` fixture to configure the `client` fixture. An example of "the modern" approach is in https://github.com/ITISFoundation/osparc-simcore/blob/136ca897d97a2e9ec2bf550f16c85c5a03ce6eeb/services/web/server/tests/unit/with_dbs/02/test_announcements.py#L35-L43

Reply:
Sorry I don't know straight away what should I change, Please show me in person, thanks!